canonical-web-and-design/ubuntu.com
# Add animation to automotive page

## Done
Add animation to the automotive page

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/automotive
- Scroll down to the "RTOS safety, Ubuntu agility" section
- Check that the animation frills and delights you

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2998
